test/compilable/vgc1.d

```diff
@@ -3,30 +3,20 @@
 
 /***************** NewExp *******************/
 
-/*
-TEST_OUTPUT:
----
-compilable/vgc1.d(17): Deprecation: class allocators have been deprecated, consider moving the allocation strategy outside of the class
-compilable/vgc1.d(18): Deprecation: class allocators have been deprecated, consider moving the allocation strategy outside of the class
----
-*/
-
 struct S1 { }
 struct S2 { this(int); }
 struct S3 { this(int) @nogc; }
-struct S4 { new(size_t); }
-struct S5 { @nogc new(size_t); }
 
 /*
 TEST_OUTPUT:
 ---
-compilable/vgc1.d(35): vgc: `new` causes a GC allocation
-compilable/vgc1.d(37): vgc: `new` causes a GC allocation
-compilable/vgc1.d(38): vgc: `new` causes a GC allocation
-compilable/vgc1.d(40): vgc: `new` causes a GC allocation
-compilable/vgc1.d(41): vgc: `new` causes a GC allocation
-compilable/vgc1.d(42): vgc: `new` causes a GC allocation
-compilable/vgc1.d(46): vgc: `new` causes a GC allocation
+compilable/vgc1.d(25): vgc: `new` causes a GC allocation
+compilable/vgc1.d(27): vgc: `new` causes a GC allocation
+compilable/vgc1.d(28): vgc: `new` causes a GC allocation
+compilable/vgc1.d(30): vgc: `new` causes a GC allocation
+compilable/vgc1.d(31): vgc: `new` causes a GC allocation
+compilable/vgc1.d(32): vgc: `new` causes a GC allocation
+compilable/vgc1.d(34): vgc: `new` causes a GC allocation
```

Comment:
Instead of changing line numbers again, can we use a `#line` directive for future-proofing ?

Reply:
Updating the test output is fully automated, so this is less work (: